data/json/items/armor.json

```
@@ -7302,6 +7305,7 @@
     "encumbrance": 50,
     "warmth": 5,
     "material_thickness": 2,
+    "flags": [ "WATERPROOF" ]
```

Looks like we're missing a comma here!

```suggestion
    "flags": [ "WATERPROOF" ],
```